docs/apply.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: layouts/apply-service-pattern.njk
```

Comment:
How comes you've created a specific layout for this page? Could it work with an existing layout?

Reply:
Not intentional. I didn't realise that I'd done so. Shall I use this one instead?
layout: layouts/service-patterns.njk

Reply:
Yes, best to use that one. That layout strips out anything unnecessary for an orphaned page.